### PR TITLE
Fixed google map issue

### PR DESCRIPTION
**Pull request by yogeshsuhagiya**
### Short description and why it's useful
- Replaced the `vue2-google-maps` library with `vue-googlemaps` package.
- Execute following commands:
 ```yarn remove -W vue2-google-maps```
 ```yarn add -W vue-googlemaps```

### CODE REVIEW

File: src/themes/default/components/core/blocks/GoogleMaps/SimpleMap.vue [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+<script>
+import { mapActions } from 'vuex'
+
+export default {
+  data () {
+    return {
+      center: {lat: -36.817685, lng: 175.699196},
+      markers: [
+        {position: {lat: -31.563910, lng: 147.154312}},
```
Comment (panoramik707) [inline src/themes/default/components/core/blocks/GoogleMaps/SimpleMap.vue]: for this version, hard-coding is fine. 
Is it the plan to have a release with dynamic lat, long @yogeshsuhagiya

File: src/themes/default/components/core/blocks/GoogleMaps/SimpleMap.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        {position: {lat: 55.822083, lng: 37.665453}},
+        {position: {lat: 55.604697, lng: 37.642107}},
+        {position: {lat: 38.749402, lng: -9.120034}},
+        {position: {lat: 38.70896, lng: -9.16913}},
+        {position: {lat: 40.784513, lng: -73.97663}},
+        {position: {lat: 40.707522, lng: -74.037055}},
+        {position: {lat: -33.890542, lng: 151.274856}},
+        {position: {lat: -33.923036, lng: 151.259052}},
+        {position: {lat: -34.028249, lng: 151.157507}},
+        {position: {lat: -33.80010128657071, lng: 151.28747820854187}},
```
Comment (panoramik707) [inline src/themes/default/components/core/blocks/GoogleMaps/SimpleMap.vue]: also I provided some lat,long as a reference which are to be used. 
#3 33°54'44.4"N 118°21'39.8"W
33.912340, -118.361061
#2 33°54'44.4"N 118°21'39.9"W
33.912341, -118.361094
#1 33°54'44.4"N 118°21'40.1"W
33.912334, -118.361126
#4 33°54'44.4"N 118°21'39.7"W
33.912333, -118.361039
#17 33°54'44.0"N 118°21'39.9"W
33.912230, -118.361094

the no after hashes are serial no. or space no.

Comment (yogeshsuhagiya) [reply src/themes/default/components/core/blocks/GoogleMaps/SimpleMap.vue]: Replace random location with given locations

File: src/themes/default/components/core/blocks/GoogleMaps/SimpleMap.vue (hunk 1 above)
Comment (yogeshsuhagiya) [reply src/themes/default/components/core/blocks/GoogleMaps/SimpleMap.vue]: Yes, there is plan to make it dynamic.

**Pull request merged**